.github/workflows/run-conventional-commits-check.yml

```diff
@@ -0,0 +1,88 @@
+name: Conventional Commit PR Titles
+
+on:
+  workflow_call:
+
+jobs:
+  validate-pr-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate PR title
+        id: validate-title
+        if: github.event.pull_request.draft == false
+        uses: actions/github-script@v8
+        with:
+          script: |
+            const pr = context.payload.pull_request;
+            const conventionalCommitFormat = /^(fix|feat|build|chore|ci|docs|style|refactor|perf|test|revert)(?:\(.+\))?!?:.+$/;
+            const titlePasses = pr.title === 'Deploy' || conventionalCommitFormat.test(pr.title);
+
+            if (titlePasses) {
+              console.log("Conventional commit check passed.");
+              return true;
+            } else {
+              console.error("Conventional commit check failed.");
+              return false;
+            }
+
+      - name: Leave comment if check failed
+        uses: actions/github-script@v8
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            if (${{steps.validate-title.outputs.result}} == false) {
```

Comment:
Ah, ok. This answers my confusion about how to pass results in from a previous step.